app.js

```diff
@@ -41,7 +41,7 @@ app.get("/v1/tokens/:token", function(req, res) {
     });
 });
 
-app.delete("/v1/tokens/delete/:token", function(req, res) {
+app.delete("/v1/tokens.json/:token", function(req, res) {
```

Comment:
This is actually a wrong way to do it but since it is what's currently being used by the example service we should proceed like that. 

The proper way to specify the format of the response is via the Accept header in the request.